src/Adapter/Http/Repository/HttpPodRepository.php

```diff
@@ -35,11 +36,18 @@ class HttpPodRepository implements PodRepository
     private $namespaceClient;
 
     /**
-     * @param HttpConnector       $connector
+     * @var PodStatusProvider
+     */
+    private $statusProvider;
```

Comment:
Is this really useful?

Reply:
Good point. That's not used now, as I kept `isTerminated()` and `isPending()` methods for backward compatibility.

Reply:
Could you remove it from here then? :)